protocol/x/clob/keeper/orders.go

```diff
@@ -241,6 +244,97 @@ func (k Keeper) PlaceShortTermOrder(
 	return orderSizeOptimisticallyFilledFromMatchingQuantums, orderStatus, err
 }
 
+// DeliverMsgCancelOrder delivers a MsgCancelOrder.
```

Comment:
Elaborate a little bit, something like this persists order cancellations on chain and sends out onchain indexer events